packages/eslint-plugin/docs/rules/explicit-module-boundary-types.md

```diff
@@ -62,14 +64,50 @@ export var arrowFn = (): string => 'test';
 export var arrowFn = (arg: string): string => `test ${arg}`;
 export var arrowFn = (arg: unknown): string => `test ${arg}`;
 
-// Class is not exported
-class Test {
-  method() {
+// A class method with no return value (void)
+export class Test {
+  method(): void {
     return;
   }
 }
 ```
 
+### 2. Non-exported members may or may not have explicit signatures
+
+<!--tabs-->
```

Comment:
I also find it confusing how this second pair of tabs has such different code between _Incorrect_ and _Correct_. And all it's trying to convey is that an alternative to adding an explicit type annotation is to not export a function.

Maybe instead of having a whole second tabs pair, a single mention could be made in the first tabs pair? I think that'd be easier to read & would result in a more streamlined docs page.